### PR TITLE
mosquitto update

### LINKED ISSUE #41: upgrade mosquitto
upgrade mosquitto image used to the latest version, ensure dyn sec plugin is available

/kind feature
/priority critical-urgent
/assign goern

### PR DESCRIPTION
**Pull request by goern**
fixed #41 

- minor corrections to keep linters happy 🙋
- :arrow_up: bounce mosquitto version to 2.0

### CODE REVIEW

File: manifests/mosquitto/deployment.yaml [hunk 1]
```diff
@@ -19,7 +23,7 @@ spec:
             - name: SOLUNAR_WS_LOG_LEVEL
               value: "1"
           name: mosquitto
-          image: docker.io/eclipse-mosquitto:1.6 
+          image: docker.io/eclipse-mosquitto@sha256:4ee2d823259ed813fc3761ac63dcf5a940231017bfb6aa4fa74615f63f1a477f
```
Comment (durandom) [inline manifests/mosquitto/deployment.yaml:26]: I'm afraid of this. It's a major version bump. Will it still work? Same config files?
Maybe try it on the local docker-compose stack first?

**Review by durandom — CHANGES_REQUESTED** [review]

File: manifests/mosquitto/deployment.yaml (hunk 1 above)
Comment (goern) [reply manifests/mosquitto/deployment.yaml:26]: I think docker composer is using v2 for quite some time

Agreed, let's test more

**Comment by goern** [comment]
this is deployed and up and running

/unhold

**Pull request merged**